M17_spec.tex

```diff
@@ -1126,31 +1126,29 @@ \section{LSF}
 	\caption{Link Setup Frame Contents}
 \end{table}
 
-\subsection{Address fields}
+\paragraph{Address fields}
 
 Destination (DST) and source (SRC) addresses may be encoded amateur radio callsigns, or special identifiers. See Address Encoding \autoref{address_encoding} for details on how up to 9 characters of text can be encoded into the 6-byte address value.
 
 The source address is always the callsign of the station transmitting, be it a client, repeater, or gateway. This is not a problem for a client, but for a repeater/gateway this raises issues about identifying the original source of a transmission. Having a repeater/gateway always use its own callsign for the source field does ensure that there are no issues with licensing authorities. To retain identification of the original source for a voice stream, an extended callsign data field will be encoded in the LSF META field.
 
 The destination address used by a client may simply be a callsign or reflector designation for a point to point contact, or may be a special identifier. Special identifiers are 6-byte addresses than can't be encoded in the standard way. For an explanation, see the Address Encoding \autoref{address_encoding}.
 
-\subsection{TYPE}
+\paragraph{TYPE}
 
 The 2-byte TYPE field contains information about the frames to follow LSF. The Packet/Stream indicator bit determines which mode (Packet or Stream) will be used during the transmission. The remaining field meanings are defined by the specific mode and application.
```

Comment:
The Packet/Stream indicator bit no longer exists, correct? So this paragraph should probably change to reflect the new scheme.